homeassistant/components/rainmachine/__init__.py

```diff
@@ -320,6 +350,19 @@ async def async_pause_watering(call: ServiceCall, controller: Controller) -> Non
         """Pause watering for a set number of seconds."""
         await controller.watering.pause_all(call.data[CONF_SECONDS])
 
+    @call_with_controller(update_programs_and_zones=False)
+    async def async_push_flow_meter_data(
+        call: ServiceCall, controller: Controller
+    ) -> None:
+        """Push flow meter data to the device."""
+        payload = {CONF_VALUE: call.data.get(CONF_VALUE)}
```

Comment:
The value is required (per the schema).

```suggestion
        payload = {CONF_VALUE: call.data[CONF_VALUE]}
```